src/test/java/lv/startup/BalticWaterTemp/BalticWaterTempApplicationTests.java

```diff
@@ -7,8 +7,5 @@
 @SpringBootTest
 class BalticWaterTempApplicationTests {
 
-	@Test
-	void contextLoads() {
-	}
 
```

Comment:
Why you removed test

Reply:
I don't see that I made this change in Github desktop